src/components/DeliverablesTable/index.tsx

```diff
@@ -41,34 +44,65 @@ const DeliverablesTable = ({
 }: DeliverablesTableProps) => {
   const dispatch = useAppDispatch();
   const { activeLocale } = useLocalization();
+  const { availableProjects: projects } = useProjects();
+  const { isAcceleratorRoute } = useAcceleratorConsole();
 
   const [deliverables, setDeliverables] = useState<ListDeliverablesElement[]>([]);
   const [deliverablesSearchRequestId, setDeliverablesSearchRequestId] = useState('');
   const deliverablesSearchRequest = useAppSelector(selectDeliverablesSearchRequest(deliverablesSearchRequestId));
 
-  const featuredFilters: FilterConfig[] = useMemo(
-    () =>
-      activeLocale
-        ? [
-            {
-              field: 'status',
-              // These options are strings for now, but may end up as enums when the BE types come through, if that is
-              // the case we will need to implement the renderOption and pillValueRenderer to render the desired
-              // human readable values
-              options: DeliverableStatuses,
-              label: strings.STATUS,
-            },
-            {
-              field: 'category',
-              // Same note as above
-              options: DeliverableCategories,
-              label: strings.CATEGORY,
-            },
-          ]
-        : [],
-    [activeLocale]
+  const getProjectName = useCallback(
```

Comment:
Any chance you can throw this in the hook?

Reply:
Sure 👍